docs/guides/guide-for-latex-users.md

```diff
@@ -670,3 +670,8 @@ applicable, contains possible workarounds.
   break will be created if this spot would be a natural page break anyways. You
   can also use `[#v(1fr)]` to distribute space on your page. It works quite
   similar to LaTeX's `\vfill`.
+
+- **Boilerplate for large scale projects.** LaTeX definitions are available in
+  included files. This is not the case in Typst, where you'll need your definitions
+  in a file of its own, and manually `import` them into each of your project's
+  `include`d files. Alternatively your Typst project can be in one very large file.
```

Comment:
This doesn't seem like the best wording for it, and an example would be nice as well. I'm not sure this belongs in "Limitations" either, since the feature does exist in Typst. Here's an initial idea on how to improve it, but larger-scale changes (e.g. moving to another section or merging with the existing text on this) may be needed.

```suggestion
- **Separation between `import` and `include`.** LaTeX macro definitions in files are
  automatically available in documents which `\include` those files' contents. In Typst,
  including a file's contents (done through `#include "file.typ"`) is separate from
  importing its definitions, which must be done with a separate
  `#import "file.typ": definitions` command. With it, you can either list the definitions
  to import individually, which is more usual when they are few (e.g.
  `#import "file.typ": x, y, template`), or, to fully mimic LaTeX's behavior, you may use a
  wildcard import to import all definitions available in another file
  (e.g. `#import "file.typ": *`).
```

Reply:
Thank you so much. I did want to get the ball rolling on this caveat because this limitation of Typst frustrates me, and I wanted it to be in the list of limitations for LaTeX users, particularly since there are at least three different issues reported by users on this, so the documentation is definitely not clear enough.

Reply:
Understood, thank you for your input. Would be nice to see what other LaTeX users think of this change as well.

Reply:
As a long-time LaTeX user, I find this very helpful. I stumbled on this a few times and, while it's not (at least from my experience) a major blocker for moving from LaTeX to Typst, making it as explicit as possible can probably make the experience even smoother than it already is.